docs/README.md

```diff
@@ -37,6 +37,14 @@ Then build the docs from source:
 $ ./build_docs.sh
 ```
 
+The shell `./build_docs.sh` will integrate external connector docs, referencing `setup_docs.sh#integrate_connector_docs`.
+This process involves cloning the repo of some external connectors, which can be time-consuming and prone to network issues.
+So, if you wish to skip this step, you can do so by adding the following arg:
```

Comment:
Nit: I think it's necessary to emphasize that  `--skip-integrate-connector-docs` option should better be picked only if the connector docs have been synced before.

Reply:
done!